test/fixtures/MeasureBundleHelpersFixtures/groupNumerAndDenomCriteria.json

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "test-measure-group",
+  "population": [
+    {
+      "id": "1b83ff0e-fcfd-48ce-9902-09be91b8350d",
+      "code": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+            "code": "initial-population",
+            "display": "Initial Population"
+          }
+        ]
+      }
+    },
+    {
+      "id": "49abc656-7d3a-44bc-92ab-6d8c40689541",
+      "code": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+            "code": "denominator",
+            "display": "Denominator"
+          }
+        ]
+      }
+    },
+    {
+      "id": "12c16634-ccf6-4d8f-acd3-5e52bdbfc5d3",
```

Comment:
Technically the denex is relevant because there's a branch of logic there, but I'll remove the numex since it isn't used.

I'll also go ahead and update the IDs to be clearer instead of random UUIDs for better readability